app/controllers/wands_controller.rb

```diff
@@ -1,8 +1,10 @@
 class WandsController < ApplicationController
   def home
```

Comment:
You can completely delete this out, because Zach made us move home to the PagesController